csp/settings.py

```diff
@@ -104,8 +102,8 @@
                 'django.contrib.messages.context_processors.messages',
             ],
             # 'loaders': [
-            # 'django.template.loaders.filesystem.Loader',
-            # 'django.template.loaders.app_directories.Loader',
+            #     'django.template.loaders.filesystem.Loader',
```

Comment:
irrelevant but if you follow PEP8 there must be only 1 space between # and the comment body